lib/NetworkManager.js

```diff
@@ -114,6 +138,29 @@ class NetworkManager extends EventEmitter {
   }
 
   /**
+   * @param {!Request} request
+   * @param {!Object} authentification challenge
+   */
+  _handleAuthenticate(request, authChallenge) {
+    request._interceptionHandled = false;
```

Comment:
let's use `this._client.send` directly to avoid messing with user-visible object.